setup.py

```diff
@@ -24,14 +24,12 @@
     install_requires=[
         "accept-types",
         "aiohttp",
-        "aiohttp_socks",
         "aiokafka",
         "avro",
         "jsonschema",
-        "kafka-python",
```

Comment:
Even if `kafka-python` gets installed as indirect dependency of aiokafka, it is also used as direct dependency e.g. admin client. I don't think it should be removed from here. And it should be also in requirements.txt?

Reply:
I'll add it back, it needs to follow the version requirements of aiokafka.

Reply:
No, I won't add it back as the kafka-python is already in the file: https://github.com/aiven/karapace/pull/448/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R19

The kafka-python is installed from a fork.